packages/travix-ui-kit/components/calendar/calendar.js

```diff
@@ -246,7 +251,7 @@ export default class Calendar extends Component {
           onMouseDown={this.handleItemMouseDown}
           onNavNextMonth={() => this.moveToMonth(CALENDAR_MOVE_TO_NEXT)}
           onNavPreviousMonth={() => this.moveToMonth(CALENDAR_MOVE_TO_PREVIOUS)}
-          onSelectDay={e => this.onSelectDay(new Date(e.currentTarget.getAttribute('data-date')))}
+          onSelectDay={e => this.onSelectDay(e.currentTarget.getAttribute('data-date'))}
```

Comment:
Any specific reason to not pass a date object to `onSelectDay` anymore?

Reply:
No any specific reason,  for me it just looks better and simpler.